api/authentication/models.py

```diff
@@ -9,7 +9,12 @@
 from django.core.mail import EmailMessage
 from django.contrib.auth.models import UserManager
 from django.contrib.auth.models import AbstractBaseUser, AbstractUser, Group
-from settings import base
+from settings.base import (
```

Comment:
its bad practice, please dont use this import in your code never. Please see in [docs example](https://docs.djangoproject.com/en/2.2/topics/settings/#using-settings-in-python-code) for import settings